addons/hr_expense/tests/test_expenses.py

```diff
@@ -417,14 +418,142 @@ def test_account_entry_multi_currency_own_account(self):
+                'date': '2022-11-19',
+                'payment_mode': 'company_account',
+                'total_amount': 1000,
+                'employee_id': self.expense_employee.id,
+                'product_id': self.product_c.id,
+                'currency_id': foreign_currency.id,  # rate is 1:1.5289
+                'tax_ids': [Command.set(taxes.ids)],
+            })],
+        })
+        sheets = expense_sheet_own_1_tax + expense_sheet_own_2_tax + expense_sheet_company_1_tax + expense_sheet_company_2_tax
+        self.assertRecordValues(sheets.mapped('expense_line_ids'), [
+            {'untaxed_amount':  869.57, 'total_amount': 1000, 'total_amount_company': 1528.90, 'amount_tax': 130.43, 'amount_tax_company': 199.42},
+            {'untaxed_amount':  769.23, 'total_amount': 1000, 'total_amount_company': 1528.90, 'amount_tax': 230.77, 'amount_tax_company': 352.82},
+            {'untaxed_amount':  869.57, 'total_amount': 1000, 'total_amount_company': 1528.90, 'amount_tax': 130.43, 'amount_tax_company': 199.42},
+            {'untaxed_amount':  769.23, 'total_amount': 1000, 'total_amount_company': 1528.90, 'amount_tax': 230.77, 'amount_tax_company': 352.82},
+        ])
+
+        sheets.action_submit_sheet()
+        sheets.approve_expense_sheets()
+        sheets.action_sheet_move_create()
+        self.assertRecordValues(expense_sheet_own_1_tax.account_move_id.mapped('line_ids'), [
+            {'balance': 1329.48, 'amount_currency': 1329.48, 'currency_id': self.company_data['currency'].id},
+            {'balance': 199.42, 'amount_currency': 199.42, 'currency_id': self.company_data['currency'].id},
+            {'balance': -1528.90, 'amount_currency': -1528.90, 'currency_id': self.company_data['currency'].id},
+        ])
+
+        self.assertRecordValues(expense_sheet_own_2_tax.account_move_id.mapped('line_ids'), [
+            {'balance': 1176.08, 'amount_currency': 1176.08, 'currency_id': self.company_data['currency'].id},
+            {'balance': 176.41, 'amount_currency': 176.41, 'currency_id': self.company_data['currency'].id},
+            {'balance': 176.41, 'amount_currency': 176.41, 'currency_id': self.company_data['currency'].id},  #  == 352.82 amount_tax_company
+            {'balance': -1528.90, 'amount_currency': -1528.90, 'currency_id': self.company_data['currency'].id},
+        ])
+
+        self.assertRecordValues(expense_sheet_company_1_tax.account_move_id.mapped('line_ids'), [
+            {'balance': 1329.48, 'amount_currency': 869.57, 'currency_id': foreign_currency.id},
+            {'balance': 199.42, 'amount_currency': 130.43, 'currency_id': foreign_currency.id},
+            {'balance': -1528.90, 'amount_currency': -1000.00, 'currency_id': foreign_currency.id},
         ])
 
+        self.assertRecordValues(expense_sheet_company_2_tax.account_move_id.mapped('line_ids'), [
+            {'balance': 1176.08, 'amount_currency': 769.23, 'currency_id': foreign_currency.id},
```

Comment:
what do you think about adding `# pylint: disable=bad-whitespace` and align keys ?